tedge/src/cli/disconnect/bridge.rs

```diff
@@ -0,0 +1,107 @@
+use crate::cli::connect::TEDGE_BRIDGE_CONF_DIR_PATH;
```

Comment:
The `disconnect` command depends on constants defined in the `connect` command. I don't think they belong here or there. Never mind. It's better to get rid of this constant here and instead inject the full path to the config file, instead of just the file name (`config_file`). Read comment below on `config_file: String`.